### ISSUE TITLE
question in data_loader.py

### ISSUE THREAD

**Issue by ybcc2015** [svishwa/crowdcount-mcnn#41]
what these two codes mean in line 34 and 35?
ht_1 = int((ht/4)*4)
wd_1 = int((wd/4)*4)

**Comment by ybcc2015** [comment]
why multiply by 4? hope to answer my doubts，thanks!

**Comment by EillotY** [comment]
i can't open the link of [Shanghai Tech A] and [Shanghai Tech B],here is my email:1203999822@qq.com,would u please send me the pre-trained model files .Thanks anyway.

**Comment by EillotY** [comment]
朋友FileNotFoundError: File b'./part_B_final/test_data/ground_truth/IMG_1.csv' does not exist
你的csv是根据步骤1，2，3，4，5来的吗  先得去matlab里生成吗 是这意思吗   
还有目前我用的参数直接是ht,和wd，因为ht_1或者wd_1这样一成就变成浮点型了在img里不适用会报错

**Comment by Cli98** [comment]
> why multiply by 4? hope to answer my doubts，thanks!

This is a trick. Adapt size to multiple of 4

**Comment by ft-z** [comment]
Did anyone has problem about 'AttributeError: 'DataFrame' object has no attribute 'as_matrix''?
有人遇到这个问题吗？在train.py调用data_loader.py的时候。

**Comment by hiramustafa77** [comment]
> Did anyone has problem about 'AttributeError: 'DataFrame' object has no attribute 'as_matrix''?
> 有人遇到这个问题吗？在train.py调用data_loader.py的时候。

yes it has been obsoleted and you may use .values instead of as_matrix()
